static/styles/settings.css

```diff
@@ -430,3 +430,13 @@ input[type=checkbox].inline-block {
 #administration #deactivation_stream_modal h3 {
     overflow-wrap: break-word;
 }
+
+.centered-footer {
+    text-align: center;
+    padding: 10px 15px;
+}
+
+#new_alias {
+    margin-right: 20px;
+    margin-bottom: auto;
+}
```

Comment:
These should be scoped inside `#administration` to avoid any risk of impacting the rest of the site.